tests/test_environment.py

```diff
@@ -74,21 +75,42 @@ def test_repr(self):
             "environment_path='path', options={})"
         )
 
-    def test_check_env_path_valid_with_valid_path(self):
-        self.environment._check_env_path_valid("valid/env/path")
+    def test_validate_path_with_valid_path(self):
+        path = self.environment._validate_path("valid/env/path")
+        assert path == "valid/env/path"
 
-    def test_check_env_path_valid_with_invalid_path_leading_slash(self):
+    def test_validate_path_with_backslashes_in_path(self):
+        path = self.environment._validate_path("valid\env\path")
+        assert path == "valid/env/path"
+
+    def test_validate_path_with_double_backslashes_in_path(self):
+        path = self.environment._validate_path('valid\\env\\path')
+        assert path == "valid/env/path"
+
+    def test_validate_path_with_leading_slash(self):
         with pytest.raises(InvalidEnvironmentPathError):
-            self.environment._check_env_path_valid(
+            self.environment._validate_path(
                 "/this/environment/path/is/invalid"
             )
 
-    def test_check_env_path_valid_with_invalid_path_trailing_slash(self):
+    def test_validate_path_with_leading_backslash(self):
+        with pytest.raises(InvalidEnvironmentPathError):
+            self.environment._validate_path(
+                '\\this\environment\path\is\invalid'
+            )
+
+    def test_validate_path_with_trailing_slash(self):
         with pytest.raises(InvalidEnvironmentPathError):
-            self.environment._check_env_path_valid(
+            self.environment._validate_path(
                 "this/environment/path/is/invalid/"
             )
 
+    def test_validate_path_with_trailing_backslash(self):
+        with pytest.raises(InvalidEnvironmentPathError):
+            self.environment._validate_path(
+                'this\environment\path\is\invalid\\'
```

Comment:
Why do we need to escape the trailing slash (if I understand what's happening here), but not the others?